lua/core/telescope.lua

```diff
@@ -159,6 +159,17 @@ function M.view_lunarvim_changelog()
   }):find()
 end
 
+function M.code_actions()
+  local opts = {
+    winblend = 10,
+    border = {},
+    borderchars = { "─", "│", "─", "│", "╭", "╮", "╯", "╰" },
+    previewer = false,
+    shorten_path = false,
+  }
+  require("telescope.builtin").lsp_code_actions(require("telescope.themes").get_dropdown(opts))
+end
+
```

Comment:
yeah but i wanted to keep it consistent and always in the middle of the screen 😅

Reply:
It's consistent until your code is under the popup 😆 
![image](https://user-images.githubusercontent.com/59826753/136097039-ddc56fd6-c929-4441-bf2f-a213aa179217.png)

I don't mind it either way since it's easy to override.

Reply:
yeah that is why some use `pumblend` to make it transparent 😄 
but yeah i think it would be easy to override so it doesn't matter

Reply:
this is when you increase the `winblend` to 20
<img width="1331" alt="Screen Shot 2021-10-06 at 12 02 28 AM" src="https://user-images.githubusercontent.com/10992695/136098365-263638d0-181b-4691-b11c-333ecde4fbc4.png">